src/shared/quota-util.h

```diff
@@ -17,3 +17,5 @@ static inline int QCMD_FIXED(uint32_t cmd, uint32_t type) {
 
 int quotactl_devnum(int cmd, dev_t devnum, int id, void *addr);
 int quotactl_path(int cmd, const char *path, int id, void *addr);
+int is_proj_id_quota_supported(int fd, uint32_t proj_id, struct dqblk *req);
+int set_proj_id_verify_exclusive_recursive(int fd, uint32_t proj_id);
```

Comment:
we usually put the subsystem name first (with some exceptions).

hence maybe name this: quota_proj_id_s_supported() and quota_proj_id_set_recursive_verify_exclusive() or so?